wedlocks/config.js

```diff
@@ -19,6 +24,18 @@ requiredVariables.forEach(envVar => {
   }
 })
 
+let wedlocksPrivateKey
+if (process.env.BASE64_WEDLOCKS_PRIVATE_KEY) {
+  // This env variable is passed as base 64 to comply with the multiline reco by circleci:
+  // https://circleci.com/docs/2.0/env-vars/#encoding-multi-line-environment-variables
+  wedlocksPrivateKey = Buffer.from(
+    process.env.BASE64_WEDLOCKS_PRIVATE_KEY,
+    'base64'
+  ).toString('utf-8')
+}
+
+console.log(wedlocksPrivateKey)
```

Comment:
Just taking another look through this and noticed this `console`. Is that intentional or did it slip through?